### PR TITLE
Exercise 3

### PR DESCRIPTION
**Pull request by marianstahl**
Hi @mmikhasenko @DhruvanshuSan! Here is a draft of the new sheet

### CODE REVIEW

File: exercises/sheet-03.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+> [!TIP]
+> Look at the [pdglive](https://pdglive.lbl.gov/Particle.action?init=0&node=S045&home=BXXX040) website of the $\Xi_c^+$ particle to get a good starting parameter for the mean of your second Gaussian.
+
+## Exercise 2
+
+Compute the profile likelihood ratio in 20 points around the best fit value of the yield. Choose the points to scan based on the uncertainties determined in Exercise 1. How does the two-sided confidence level at 68.3% (1 $\sigma$) constructed with the profile likelihood compare to the uncertainty calculated from the Hessian matrix?
+
+## Exercise 3
+
+Use the jackknife bootstrapping method to estimate the variance of the yield. How does it compare to the previous methods?
```
Comment (mmikhasenko) [inline exercises/sheet-03.md]: two methods:
- jackknife, 
- bootstrapping

File: exercises/sheet-03.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+You will continue working with the sample that you have obtained for Sheet 2.
+
+When fitting the $\Xi^- K^+ \pi^+$ invariant-mass distribution (`Lc_M`) for the previous sheet and inspecting the pulls carefully, you may have noticed a second peak at higher mass.<br>
+This is exciting! If you can prove that the small enhancement is statistically significant, you have evidence for a decay process that has never been measured: $\Xi_c^+ \rightarrow \Xi^- K^+ \pi^+$.<br>
+To work towards that, you will estimate the *yield* (i.e. the integral of the signal p.d.f.) of this process.
+
+## Exercise 1
+
+Add a second Gaussian to your fit model, fit the data again, and compute the Hessian matrix. Use that to compute the uncertainty of the signal yield for the new decay process.
```
Comment (mmikhasenko) [inline exercises/sheet-03.md]: Ask also about parameter correlation. I suggest asking to plot the correlation matrix in color coding `heatmap`

Comment (mmikhasenko) [inline exercises/sheet-03.md]: We should try if fit is possible, of the signals runs away. Maybe some guidance on constrained fit is needed

**Comment by mmikhasenko** [comment]
@marianstahl the problems are great.
I left some comments. As soon as we update the reference code for fitting, students will be good to go and solve the sheet

File: exercises/sheet-03.md (hunk 2 above)
Comment (marianstahl) [reply exercises/sheet-03.md]: Did you discuss constrained fits in the lecture yet? If not, I think the concept is not too complicated, and it's maybe enough if we point to the julia documentation on this and explain that the mass of the decaying particle is known and that the width of the Gaussian should be up to 50 % wider.

**Comment by marianstahl** [comment]
> @marianstahl the problems are great. I left some comments. As soon as we update the reference code for fitting, students will be good to go and solve the sheet

I think we cannot put the reference code for fitting in the repo just yet, if we allow that sheet 2 can be handed in until next week. The new sheet could be released nevertheless I would say

**Pull request merged**